R/bootstrap-layout.R

```diff
@@ -28,6 +28,8 @@
 #'   Shiny-Application-Layout-Guide](http://shiny.rstudio.com/articles/layout-guide.html) for additional details on laying out fluid
 #'   pages.
 #'
+#' @family layout functions
+#'
 #' @seealso [column()], [sidebarLayout()]
```

Comment:
Since `sidebarLayout()` is part of the layout family, it should be removed from this `@seealso` section